<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SummaryPage</class>
 <widget class="QWidget" name="SummaryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Summary</string>
  </property>
  <property name="styleSheet">
   <string notr="true">SummaryPage {
    background-color: #e8f5e9;
}

QGroupBox {
    background-color: white;
    border: 1px solid #c8e6c9;
    border-radius: 8px;
    margin-top: 10px;
    padding: 10px;
    font-weight: bold;
}

QRadioButton {
    color: black;
    spacing: 8px;
}

QRadioButton::indicator {
    width: 18px;
    height: 18px;
}

QRadioButton::indicator:checked {
    background-color: #43a047;
    border: 2px solid #c8e6c9;
    border-radius: 9px;
}

QRadioButton::indicator:unchecked {
    background-color: white;
    border: 2px solid #c8e6c9;
    border-radius: 9px;
}

QComboBox {
    background-color: #f5f5f5;
    border: 1px solid #c8e6c9;
    border-radius: 4px;
    padding: 6px;
    color: black;
    min-height: 30px;
}

QComboBox::drop-down {
    border: none;
}

QComboBox::down-arrow {
    image: none;
    border-left: 5px solid transparent;
    border-right: 5px solid transparent;
    border-top: 5px solid #43a047;
    margin-right: 5px;
}

QListWidget {
    background-color: white;
    border: 1px solid #81c784;
    border-radius: 8px;
    color: black;
}

QPushButton {
    background-color: #43a047;
    color: white;
    font-weight: bold;
    border-radius: 6px;
    padding: 10px 20px;
    min-height: 40px;
}

QPushButton:hover {
    background-color: #2e7d32;
}

QLabel {
    color: black;
}

QTextEdit {
    background-color: white;
    border: 1px solid #81c784;
    border-radius: 8px;
    color: black;
    padding: 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="homeBtn">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #43a047;
    color: white;
    font-size: 20px;
    border-radius: 6px;
    padding: 5px;
}
QPushButton:hover {
    background-color: #2e7d32;
}</string>
       </property>
       <property name="text">
        <string>üè†</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;</string>
       </property>
       <property name="text">
        <string>üí∞ Financial Summary</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="viewTypeGroup">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="title">
      <string>View By</string>
     </property>
     <layout class="QHBoxLayout" name="viewTypeLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QRadioButton" name="goalRadioBtn">
        <property name="text">
         <string>üéØ Goal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="categoryRadioBtn">
        <property name="text">
         <string>üìÅ Category</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="selectionGroup">
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QVBoxLayout" name="selectionLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="selectionLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select Goal:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selectionComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>44</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeFrameLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Time Frame:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="timeFrameComboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>44</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Daily</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monthly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yearly</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="generateBtn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #43a047;</string>
     </property>
     <property name="text">
      <string>üìà Generate Summary</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
